test/native/setup-after-env.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * WordPress dependencies
+ */
+import '@wordpress/jest-console';
```

Comment:
Now that we import this module for all tests, we can consider removing the import from test cases where it was added:

https://github.com/WordPress/gutenberg/blob/c8b392a8404889b296e02ed9e8f30ae77a3bd6f4/packages/block-editor/src/components/block-draggable/test/index.native.js#L19
https://github.com/WordPress/gutenberg/blob/c8b392a8404889b296e02ed9e8f30ae77a3bd6f4/packages/block-library/src/image/test/edit.native.js#L29
https://github.com/WordPress/gutenberg/blob/c8b392a8404889b296e02ed9e8f30ae77a3bd6f4/packages/react-native-editor/src/test/index.test.js#L14
https://github.com/WordPress/gutenberg/blob/c8b392a8404889b296e02ed9e8f30ae77a3bd6f4/packages/rich-text/src/test/index.native.js#L18

Reply:
Address in c28087f5afd8530076aa8ffa16813b9cd3706d63.